test/unit/repo_labels_assigner_test.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RepoLabelsAssignerTest < ActiveSupport::TestCase
```

Comment:
Outside of my comfort zone here (I'm used to RSpec) so if this needs to change please let me know!